indico/modules/events/contributions/controllers.py

```diff
@@ -95,8 +79,40 @@ class RHContributions(RHManageContributionsBase):
     """Display contributions management page"""
 
     def _process(self):
+        contrib_report_config = reporter.get_report_config(self.event_new.id)
+        if 'config' in request.args:
+            return redirect(url_for('.manage_contributions', self.event_new))
+
+        contrib_report_args = reporter.get_contrib_report_args(self.event_new, contrib_report_config['filters'])
+
         return WPManageContributions.render_template('management/contributions.html', self._conf, event=self.event_new,
-                                                     **_get_contribution_list_args(self.event_new))
+                                                     **contrib_report_args)
+
+
+class RHContributionsReportCustomize(RHManageContributionsBase):
+    """Filter options for the contributions report of an event"""
+
+    def _process_GET(self):
+        contrib_report_config = reporter.get_report_config(self.event_new.id)
```

Comment:
I think `report_config` is a better name. We won't have more reports on this RH, anyway.